packages/core/admin/admin/src/content-manager/components/DynamicTable/BulkActionsBar/index.js

```diff
@@ -1,31 +1,164 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-import { Button } from '@strapi/design-system';
-import { useTracking } from '@strapi/helper-plugin';
+import { Button, Flex, Typography } from '@strapi/design-system';
+import { Check, ExclamationMarkCircle, Trash } from '@strapi/icons';
 import { useIntl } from 'react-intl';
+import { useTracking } from '@strapi/helper-plugin';
+import { getTrad } from '../../../utils';
+import InjectionZoneList from '../../InjectionZoneList';
+import ConfirmBulkActionDialog from '../ConfirmBulkActionDialog';
+
+const confirmDialogsPropTypes = {
+  isConfirmButtonLoading: PropTypes.bool.isRequired,
+  isOpen: PropTypes.bool.isRequired,
+  onConfirm: PropTypes.func.isRequired,
+  onToggleDialog: PropTypes.func.isRequired,
+};
+
+const ConfirmDialogPublishAll = ({ isOpen, onToggleDialog, isConfirmButtonLoading, onConfirm }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <ConfirmBulkActionDialog
+      isOpen={isOpen}
+      onToggleDialog={onToggleDialog}
+      icon={<ExclamationMarkCircle />}
```

Comment:
If they all use the same icon can we just get rid of this prop and keep the icon in the ConfirmBulkActionDialog?